web-ui/src/main/resources/catalog/components/viewer/gfi/FeaturesLoader.js

```diff
@@ -264,8 +264,8 @@
         for (var i = 0; i < columns.length; i++) {
           var fieldSpec = dictionary[columns[i]["field"]];
           if (angular.isDefined(fieldSpec)) {
-            columns[i]["title"] = fieldSpec.name;
-            columns[i]["titleTooltip"] = fieldSpec.definition || fieldSpec.code;
+            columns[i]["title"] = fieldSpec.code;
```

Comment:
We want to display the column name (instead of code) and definition as tooltip no ?
cf. https://github.com/geonetwork/core-geonetwork/blob/main/schemas/iso19110/src/main/plugin/iso19110/index-fields/index.xsl#L97-L98 and https://github.com/geonetwork/core-geonetwork/blob/main/schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/index-fields/index.xsl#L1001-L1003

Reply:
But the `fieldSpec.code` seems the name and the `fieldSpec.name` the definition for the tooltip. According to this, there is no `fieldSpec.definition` information:

https://github.com/geonetwork/core-geonetwork/blob/37067f9c169f9e7d01f0cbb8c2ab6857536f4ceb/web-ui/src/main/resources/catalog/components/viewer/gfi/FeaturesTableService.js#L66-L69

Reply:
Ah yes in this case, so it is not consistent when we have an ISO19110 feature catalog and an embedded one. See https://github.com/geonetwork/core-geonetwork/blob/main/schemas/iso19110/src/main/plugin/iso19110/index-fields/index.xsl#L97-L98 and https://github.com/geonetwork/core-geonetwork/blob/main/services/src/main/java/org/fao/geonet/api/records/MetadataApi.java#L637-L647.

Maybe better to rename `code` by `name` and `name` by `definition` in the `dictionary` var and then it should work in both case.